digdag-standards/src/main/java/io/digdag/standards/operator/gcp/BqClient.java

```diff
@@ -170,11 +170,13 @@ void submitJob(String projectId, Job job)
                 .execute();
     }
 
-    Job jobStatus(String projectId, String jobId)
+    Job jobStatus(String projectId, String jobId, String location)
             throws IOException
     {
         return client.jobs()
                 .get(projectId, jobId)
+                // newer version of google-api-services-bigquery-v2 has setLocation()
+                .set("location", location)
```

Comment:
I think calling only when `location` is present is better.

Reply:
Done.